content/dpg indicators/1/_index.en.adoc

```diff
@@ -0,0 +1,8 @@
+---
+title: "1. Relevance to Sustainable Development Goals"
+icon: "ti-anchor"
+description: "Digital public goods must be designed and developed to advance the Sustainable Development Goals (SDGs) and demonstrate such by providing links/documentation that support their relevance. "
+type: "indicator"
+
+---
+
```

Comment:
Note that the file path for all the categories should not include whitespace (i.e. `/dpg indicators/` should be `/dpg-indicators/`. Otherwise, the browser will insert escape characters into the URL for the whitespace and this can impact search engine optimization.

Reply:
Sorry, I somehow skipped this mail by mistake.
Yeah i will change it👍

Reply:
I have fixed this in the new repo.